receiver/hostmetricsreceiver/README.md

```diff
@@ -38,3 +44,56 @@ service:
     metrics:
       receivers: [hostmetrics, hostmetrics/disk]
 ```
+
+## Scrapers
+
+The available scrapers are:
```

Comment:
nit: the corresponding scraper names in the config are not documented, e.g. 
```markdown
- **CPU** (`cpu`): ...
```

Might be easier to read if converted to table.